float32 x_accel
float32 y_accel
float32 z_accel
float32 x_gyro
float32 y_gyro
float32 z_gyro
float32 x_mag
float32 y_mag
float32 z_mag
